doc/howToBuildADuck.md

```diff
@@ -320,6 +329,14 @@ The cause of this error because the library is missing. To install the library,
 
 This issue occurred because when the program tried to compile, it came across two “Wifi.h” files and was not sure which one to choose. The Wifi.h that is needed is 
 
+#### Linux issue with permission denied to port 
+* ser_open(): can't open device "/dev/ttyACM0": Permission denied
+* To fix this run the following commands via terminal 
+$ sudo usermod -a -G dialout <username>
+$ sudo chmod a+rw /dev/ttyACM0
+* Replace username with your devices name & Replace /dev/ttyACM0 with your USB Port
+* That can be found inside Arduino IDE via TOOLS > Port 
```

Comment:
Even this is correct it is highly Ubuntu depended. To be more generic: The user needs to be in the group who has access to the device. With `stats /dev/tty<WHATEVER>` you could find the group. Than add the current user to that group.

Reply:
yes, that part could use some improvement. 
* replace the string "username" with "devicename" in both command example and description.
* the "usermod" and "chmod" parts are redundant. if you add yourself to the group, you dont need the chmod, if you do the chmod (which gives access to the device to everyone) you dont need the usermod.
* i am ok with using the groupname "dialout" in the example because thats what the group is called in the four flavors of linux i just checked, but adding a note along the lines of using stats or ls -l to check certainly wouldnt hurt.

perhaps the title should be changed to "ubuntu" instead of "linux" for this part?

Reply:
maybe all this Linux stuff fits better into a FAQ than in a README